bin/fiwareDeviceSimulatorCLI

```diff
@@ -326,12 +327,40 @@ function updateScheduledJobsInGoogleSheet() {
   }
 }
 
+/**
+ * Process the elements of the simulation JSON (starting by the root element in a recursive way) and replace
+ * each ${...} token by the corresponding environment variable value (supposedly defined in .env file, otherwise
+ * the token is replaced by a blank).
+ */
+function applyEnvVars(file) {
+  const result = file;
+  for (const key in result) {
+    const element = result[key];
+    if (typeof element === 'object' && element.length === undefined) {
+      file[key] = applyEnvVars(element);
+    } else if (typeof element === 'string') {
+      const match = element.match(/^\${[A-Za-z_-]+}/g);
+      if (match && match.length > 0) {
+        let command = match[0].replace('${', '').replace('}', '');
+        command = `process.env.${command}`;
+        result[key] = eval(command) ? eval(command) : '';
+      }
+    } else if (typeof element === 'object' && element.length > 0) {
+      element.forEach((item, index) => {
+        element[index] = applyEnvVars(item);
+      });
```

Comment:
The if clause has "three" branches in this order

* For object (recursive)
* For string
* For arrays (recursive)

From a style point of view I think is better to group the two ones with recursivity first, I mean:

```
* For object (recursive)
* For arrays (recursive)
* For string
```

Reply:
FIXED